extensions/amp-skimlinks/0.1/affiliate-link-resolver.js

```diff
@@ -14,55 +16,134 @@ export const STATUS__IGNORE_LINK = 'ignore';
 // 'can be monetized' until we have the answer from the API.
 export const STATUS__UNKNOWN = 'unknown';
 
-export default class AffiliateLinkResolver {
+/**
+ * The AffiliateLinkResolver class is in charge of "resolving"
+ * links, namely, determining if the url from a link can be replaced by
+ * a new Skimlinks monetisable url.
```

Comment:
`monetizable` 💅 

To be fair, it seems that both ways (z or s) work. Just raising it as it was used with z in other files 🙈